make/langtools/netbeans/nb-javac/build.xml

```diff
@@ -166,6 +166,11 @@
             <arg value="${src.dir}/java.compiler/share/classes"/>
             <arg value="${src.dir}/jdk.compiler/share/classes"/>
         </java>
+        <exec failonerror="true" executable="patch" dir="${src.dir}">
```

Comment:
- I don't want to accept (new) patches to this repository!
- I would rather see a Jackpot rule to rewrite `_` to something else
- or even better a compiler that can handle `_`

Reply:
unfortunately jackpot can't match `_` atm since it is a synthetic path (and no real variable token). I tried that already.

Reply:
I see. The 38bb91e however seems good enough.